css/style.css

```diff
@@ -1,3 +1,6 @@
+body{
```

Comment:
Yeah this is confusing, because this file is called style.css, I would assume it is being applied everywhere, but it is only for index page..

Because also you are not using classnames it is all a bit confused.

Maybe rename this file `landing`?